src/app/App.tsx

```diff
@@ -135,6 +140,10 @@ const Main = () => {
 
   useEffect(() => {
     if (isLoggedIn) {
+      if (shouldDisplayNewHomeView) {
+        prefetchUrl("/")
+      }
```

Comment:
To avoid nested if statements and make this easier to read:
```suggestion
    if (isLoggedIn && shouldDisplayNewHomeView) {
      prefetchUrl("/")
```

Reply:
nice idea but I would have to check again for `isLoggedIn` since that's important for line 147 where we call `savePendingToken`

Do you mean something like

```typescript
if (isLoggedIn && shouldDisplayNewHomeView) {
	 prefetchUrl("/")
}
if (isLoggedIn) {
	 savePendingToken()
}
```

Reply:
oh, I haven't that there is more in the first if-statement...